language_detector/languages.py

```diff
@@ -6,4 +6,26 @@
         'common_words': ['yo', 'tu', 'el', 'nosotros']  # add your words here
     },
     #  include your extra languages here
+    
+    
+    {
+        'name': 'German',
+        'common_words' : [
+                    'das', 'ist', 'du', 'ich', 'nicht', 'die', 'es', 'und',
+                    'der', 'was', 'wir', 'zu', 'ein', 'er', 'in', 'sie', 'mir',
+                    'mit', 'ja', 'wie', 'den', 'auf', 'mich', 'dass', 'so',
+                    'hier', 'eine', 'wenn', 'hat', 'all', 'sind', 'von',
+                    'dich', 'war', 'haben', 'für', 'an', 'habe', 'da', 'nein',
```

Comment:
The word `für` is breaking your tests on Python 2.7. Include a line like this one: https://github.com/rmotr-group-projects/pyp-w1-gw-language-detector/blob/master/tests/test_main.py#L1